app/src/lib/auth.ts

```diff
@@ -75,7 +75,7 @@ export const authOptions: NextAuthOptions = {
           }
           const users = await db.models.User.findAll();
           user = await db.models.User.findOne({
-            where: { email: credentials.email },
+            where: { email: credentials.email.toLowerCase() },
```

Comment:
Are we sure about this change? I get that email addresses should be context sensitive (usually), just double checking.